custom_model_runner/datarobot_drum/drum/main.py

```diff
@@ -86,9 +83,15 @@ def signal_handler(sig, frame):
 
         options = arg_parser.parse_args()
         CMRunnerArgsRegistry.verify_options(options)
-
         runtime.options = options
 
+        # mlpiper restful_component relies on SIGINT to shutdown nginx and uwsgi,
+        # so we don't intercept it.
```

Comment:
Does this have any adverse affects? what was it being used for before?

Reply:
Looks like it has been used only in one case:
running  in server mode locally in docker. In this case, when hitting Ctrl-C it would call `shutdown` API on the server, drum will shop and exit, and container will be stopped.

Ctrl+C is not relevant to the drop-in env.
And I didn't had any problems when running in server production mode in docker locally.

But this `in_perf_mode_internal` is confusing. I'll try to clean it.

Reply:
Thanks

Reply:
**DONE**